tests/karma/unit/services/enketo.js

```diff
@@ -367,6 +373,210 @@ describe('Enketo service', function() {
       });
     });
 
+    it('creates extra docs', function() {
+      /* jshint expr: true */
+
+      form.validate.returns(KarmaUtils.mockPromise(null, true));
+      var content =
```

Comment:
Use ES6 multiline strings - a backtick at the start and end and all the concatenation goes away.